node/runtime/src/lib.rs

```diff
@@ -82,7 +82,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	// implementation changes and behavior does not, then leave spec_version as
 	// is and increment impl_version.
 	spec_version: 190,
-	impl_version: 190,
+	impl_version: 191,
```

Comment:
Since this change actually alters behavior of the runtime it looks like the `spec_version` should be bumped as well.